arm/Microsoft.Network/vpnGateways/.parameters/parameters.json

```diff
@@ -3,10 +3,10 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "name": {
-            "value": "carml-az-vpngw-x-001"
+            "value": "<<namePrefix>>-az-vpngw-x-001"
         },
         "virtualHubResourceId": {
-            "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/virtualHubs/SampleVirtualHub"
+            "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/virtualHubs/<<namePrefix>>-az-vhub-x-001"
```

Comment:
Please update accordingly also the references to "sampleVirtualHub" and "sampleVPNsite" below

Reply:
done